ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCUI-prod.json

```diff
@@ -100,5 +100,47 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
```

Comment:
I am not sure we need this event in the prod environment

Reply:
I think we need to remove from the -prod.json.
@hmcts-version1-domini

Reply:
Done. Thank you both!